receiver/vcenterreceiver/README.md

```diff
@@ -0,0 +1,57 @@
+# vCenter Receiver
+
+This receiver fetches metrics from a vCenter or ESXi host running VMware vSphere APIs.
+
+Supported pipeline types: `metrics` and `logs`
+
+> :construction: This receiver is in **BETA**. Configuration fields and metric data model are subject to change.
+
+## Prerequisites
+
+This receiver has been built to support ESXi and vCenter versions:
+
+- 7.0
+- 6.7
+- 7.5
+
+A “Read Only” user assigned to a vSphere with permissions to the vCenter server, cluster and all subsequent resources being monitored. Must be specified in order for the receiver to retrieve information about them.
+
+## Configuration
+
+Configuration is split up into `metrics` and `logging`
+
+```yaml
+receivers:
+    vcenter:
+      metrics:
+      logging:
+```
+
+### Metrics Configuration
+
+| Parameter | Default | Type | Notes |
+| --- | --- | --- | --- |
+| endpoint |  | String | Endpoint to the vCenter Server or ESXi host that has the sdk path enabled. Required. The expected format is `<protocol>://<hostname>` <br><br> i.e: `https://vcsa.hostname.localnet` |
+| username |  | String | Required |
+| password |  | String | Required |
+| tls | | TLSClientSetting | Not Required. Will use defaults for [configtls.TLSClientSetting](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md). By default insecure settings are rejected and certificate verification is on. |
```

Comment:
Depending on the size of vCenter it may take longer to do the collection. Of my testing I think we can tone it back to 1 or 2 minutes and be relatively safe!

Reply:
Ok, please explain in the docs.